tests/integration-tests/clusters_factory.py

```diff
@@ -86,6 +87,11 @@ def create_cluster(self, cluster):
             raise Exception(error)
         logging.info("Cluster {0} created successfully".format(name))
 
+        # FIXME: temporary workaround since in certain circumstances the cluster isn't ready for
+        # job submission right after creation. We need to investigate this further.
+        logging.info("Sleeping for 60 seconds in case cluster is not ready yet".format(name))
```

Comment:
I think you are missing the `{0}`, or you should remove the `format`

Reply:
Removed the format. Thanks!